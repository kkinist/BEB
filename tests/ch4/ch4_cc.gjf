%chk=ch4.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 6: CCSD(T) for neutral ch4

0 1

C H 0
cc-pVTZ
****

