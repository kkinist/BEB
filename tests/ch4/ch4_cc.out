 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-28683.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     28685.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch4.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -----------------------------------
 BEB step 6: CCSD(T) for neutral ch4
 -----------------------------------
 Structure from the checkpoint file:  "ch4.chk"
 Charge =  0 Multiplicity = 1
 Redundant internal coordinates found in file.
 C,0,0.,0.000000001,0.000000001
 H,0,-0.000000006,-0.0000000007,1.0935113774
 H,0,0.,-1.0309724112,-0.3645037927
 H,0,-0.892848297,0.5154862096,-0.3645037952
 H,0,0.892848303,0.5154862062,-0.3645037855
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    0.000000    1.093511
      3          1           0        0.000000   -1.030972   -0.364504
      4          1           0       -0.892848    0.515486   -0.364504
      5          1           0        0.892848    0.515486   -0.364504
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.093511   0.000000
     3  H    1.093511   1.785697   0.000000
     4  H    1.093511   1.785697   1.785697   0.000000
     5  H    1.093511   1.785697   1.785697   1.785697   0.000000
 Stoichiometry    CH4
 Framework group  TD[O(C),4C3(H)]
 Deg. of freedom     1
 Full point group                 TD      NOp  24
 Largest Abelian subgroup         D2      NOp   4
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.631339    0.631339    0.631339
      3          1           0       -0.631339   -0.631339    0.631339
      4          1           0       -0.631339    0.631339   -0.631339
      5          1           0        0.631339   -0.631339   -0.631339
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    157.2594344    157.2594344    157.2594344
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      5 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    26 symmetry adapted cartesian basis functions of A   symmetry.
 There are    23 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    23 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    23 symmetry adapted cartesian basis functions of B3  symmetry.
 There are    23 symmetry adapted basis functions of A   symmetry.
 There are    21 symmetry adapted basis functions of B1  symmetry.
 There are    21 symmetry adapted basis functions of B2  symmetry.
 There are    21 symmetry adapted basis functions of B3  symmetry.
    86 basis functions,   120 primitive gaussians,    95 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy        13.3922467928 Hartrees.
 NAtoms=    5 NActive=    5 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    86 RedAO= T EigKep=  1.58D-03  NBF=    23    21    21    21
 NBsUse=    86 1.00D-06 EigRej= -1.00D+00 NBFU=    23    21    21    21
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (A1) (A1) (T2) (T2) (T2)
       Virtual   (A1) (T2) (T2) (T2) (T2) (T2) (T2) (T2) (T2) (T2)
                 (A1) (E) (E) (A1) (T1) (T1) (T1) (T2) (T2) (T2)
                 (T2) (T2) (T2) (T2) (T2) (T2) (E) (E) (A1) (T2)
                 (T2) (T2) (A1) (T2) (T2) (T2) (E) (E) (T1) (T1)
                 (T1) (T2) (T2) (T2) (T1) (T1) (T1) (T2) (T2) (T2)
                 (E) (E) (A1) (T2) (T2) (T2) (E) (E) (T1) (T1)
                 (T1) (T2) (T2) (T2) (A1) (T2) (T2) (T2) (T2) (T2)
                 (T2) (T1) (T1) (T1) (A1) (E) (E) (T2) (T2) (T2)
                 (A1)
 The electronic state of the initial guess is 1-A1.
 Keep R1 ints in memory in symmetry-blocked form, NReq=9019840.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RHF) =  -40.2131230973     A.U. after    8 cycles
            NFock=  8  Conv=0.54D-08     -V/T= 2.0021
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    86
 NBasis=    86 NAE=     5 NBE=     5 NFC=     1 NFV=     0
 NROrb=     85 NOA=     4 NOB=     4 NVA=    81 NVB=    81

 **** Warning!!: The largest alpha MO coefficient is  0.12643695D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048223117
 LASXX=       294099 LTotXX=      294099 LenRXX=      598546
 LTotAB=      304447 MaxLAS=      613020 LenRXY=           0
 NonZer=      892645 LenScr=     2007552 LnRSAI=      613020
 LnScr1=     1571840 LExtra=           0 Total=      4790958
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=0 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.6116534380D-02 E2=     -0.1811401775D-01
     alpha-beta  T2 =       0.4900839824D-01 E2=     -0.1621472128D+00
     beta-beta   T2 =       0.6116534380D-02 E2=     -0.1811401775D-01
 ANorm=    0.1030165747D+01
 E2 =    -0.1983752483D+00 EUMP2 =    -0.40411498345640D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=8950725.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 MP4(R+Q)=  0.39246737D-02
 E3=       -0.18300296D-01        EUMP3=      -0.40429798641D+02
 E4(DQ)=   -0.93154115D-03        UMP4(DQ)=   -0.40430730183D+02
 E4(SDQ)=  -0.19262266D-02        UMP4(SDQ)=  -0.40431724868D+02
 DE(Corr)= -0.21266434     E(Corr)=     -40.425787436    
 NORM(A)=   0.10364656D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21740738     E(CORR)=     -40.430530481     Delta=-4.74D-03
 NORM(A)=   0.10386440D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21831387     E(CORR)=     -40.431436965     Delta=-9.06D-04
 NORM(A)=   0.10390233D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21859436     E(CORR)=     -40.431717453     Delta=-2.80D-04
 NORM(A)=   0.10390770D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21861354     E(CORR)=     -40.431736637     Delta=-1.92D-05
 NORM(A)=   0.10390786D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21861413     E(CORR)=     -40.431737223     Delta=-5.86D-07
 NORM(A)=   0.10390777D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21861404     E(CORR)=     -40.431737140     Delta= 8.32D-08
 NORM(A)=   0.10390779D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        20
 NAB=    10 NAA=     0 NBB=     0.
 DE(Corr)= -0.21861409     E(CORR)=     -40.431737183     Delta=-4.36D-08
 NORM(A)=   0.10390779D+01
 Largest amplitude= 2.41D-02
 Time for triples=        3.57 seconds.
 T4(CCSD)= -0.65101780D-02
 T5(CCSD)=  0.19860914D-03
 CCSD(T)= -0.40438048752D+02
 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A1) (A1) (T2) (T2) (T2)
       Virtual   (A1) (T2) (T2) (T2) (T2) (T2) (T2) (A1) (T2) (T2)
                 (T2) (E) (E) (A1) (T1) (T1) (T1) (T2) (T2) (T2)
                 (T2) (T2) (T2) (T2) (T2) (T2) (E) (E) (A1) (T2)
                 (T2) (T2) (A1) (T2) (T2) (T2) (T1) (T1) (T1) (E)
                 (E) (T2) (T2) (T2) (T1) (T1) (T1) (T2) (T2) (T2)
                 (E) (E) (A1) (T2) (T2) (T2) (E) (E) (T1) (T1)
                 (T1) (T2) (T2) (T2) (A1) (T2) (T2) (T2) (T2) (T2)
                 (T2) (T1) (T1) (T1) (A1) (E) (E) (T2) (T2) (T2)
                 (A1)
 The electronic state is 1-A1.
 Alpha  occ. eigenvalues --  -11.20901  -0.94097  -0.54363  -0.54363  -0.54363
 Alpha virt. eigenvalues --    0.14316   0.22895   0.22895   0.22895   0.36461
 Alpha virt. eigenvalues --    0.36461   0.36461   0.63979   0.64578   0.64578
 Alpha virt. eigenvalues --    0.64578   0.68359   0.68359   0.75493   1.07333
 Alpha virt. eigenvalues --    1.07333   1.07333   1.15835   1.15835   1.15835
 Alpha virt. eigenvalues --    1.31782   1.31782   1.31782   1.53312   1.53312
 Alpha virt. eigenvalues --    1.53312   1.59676   1.59676   1.79916   2.42685
 Alpha virt. eigenvalues --    2.42685   2.42685   2.56813   2.98809   2.98809
 Alpha virt. eigenvalues --    2.98809   3.13594   3.13594   3.13594   3.13811
 Alpha virt. eigenvalues --    3.13811   3.30650   3.30650   3.30650   3.61498
 Alpha virt. eigenvalues --    3.61498   3.61498   3.73182   3.73182   3.73182
 Alpha virt. eigenvalues --    3.79774   3.79774   3.86841   4.09456   4.09456
 Alpha virt. eigenvalues --    4.09456   4.12127   4.12127   4.37876   4.37876
 Alpha virt. eigenvalues --    4.37876   4.52492   4.52492   4.52492   4.82193
 Alpha virt. eigenvalues --    5.04360   5.04360   5.04360   5.47843   5.47843
 Alpha virt. eigenvalues --    5.47843   5.49849   5.49849   5.49849   5.55223
 Alpha virt. eigenvalues --    5.62170   5.62170   6.20562   6.20562   6.20562
 Alpha virt. eigenvalues --   14.39867
          Condensed to atoms (all electrons):
               1          2          3          4          5
     1  C    4.729755   0.408543   0.408543   0.408543   0.408543
     2  H    0.408543   0.586062  -0.028529  -0.028529  -0.028529
     3  H    0.408543  -0.028529   0.586062  -0.028529  -0.028529
     4  H    0.408543  -0.028529  -0.028529   0.586062  -0.028529
     5  H    0.408543  -0.028529  -0.028529  -0.028529   0.586062
 Mulliken charges:
               1
     1  C   -0.363926
     2  H    0.090982
     3  H    0.090982
     4  H    0.090982
     5  H    0.090982
 Sum of Mulliken charges =   0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.000000
 Electronic spatial extent (au):  <R**2>=             35.9056
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -8.4401   YY=             -8.4401   ZZ=             -8.4401
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              0.0000   YY=              0.0000   ZZ=              0.0000
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.7095
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -16.7978 YYYY=            -16.7978 ZZZZ=            -16.7978 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -5.0556 XXZZ=             -5.0556 YYZZ=             -5.0556
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 1.339224679282D+01 E-N=-1.197783110610D+02  KE= 4.012933022383D+01
 Symmetry A    KE= 3.445027374864D+01
 Symmetry B1   KE= 1.893018825064D+00
 Symmetry B2   KE= 1.893018825064D+00
 Symmetry B3   KE= 1.893018825064D+00
 1\1\GINC-GAMBA\SP\RCCSD(T)-FC\Gen\C1H4\IRIKURA\17-May-2017\0\\# CCSD(T
 ,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 6: CCSD(T) for n
 eutral ch4\\0,1\C,0,0.,0.000000001,0.000000001\H,0,-0.000000006,-0.000
 0000007,1.0935113774\H,0,0.,-1.0309724112,-0.3645037927\H,0,-0.8928482
 97,0.5154862096,-0.3645037952\H,0,0.892848303,0.5154862062,-0.36450378
 55\\Version=ES64L-G09RevD.01\State=1-A1\HF=-40.2131231\MP2=-40.4114983
 \MP3=-40.4297986\MP4D=-40.4346549\MP4DQ=-40.4307302\MP4SDQ=-40.4317249
 \CCSD=-40.4317372\CCSD(T)=-40.4380488\RMSD=5.398e-09\PG=TD [O(C1),4C3(
 H1)]\\@


 THE WORLD IS NOT A HAMBURGER - YOU CAN'T HAVE IT YOUR WAY.
 Job cpu time:       0 days  0 hours  0 minutes 28.6 seconds.
 File lengths (MBytes):  RWF=     45 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:20:58 2017.
